app/code/Magento/Inventory/composer.json

```diff
@@ -4,6 +4,8 @@
     "require": {
         "php": "7.0.2|7.0.4|~7.0.6|~7.1.0",
         "magento/framework": "100.3.*",
+        "magento/module-catalog": "100.0.*",
```

Comment:
Inventory is our base module
So Inventory MUST NOT have dependencies on `Catalog`,  `CatalogInventory` or `InventoryCatalog`